lib/datasources/location.ts

```diff
@@ -1,19 +1,22 @@
-import type { Location } from "@/schema";
+import type { Geofence, Location } from "@/schema";
 import type { WithKey } from "@/util";
 import Dataloader from "dataloader";
 import type { Request } from "express";
 import { GraphQLError } from "graphql/error";
 import { sql } from "./postgres";
 
 export default (_: Request) =>
-  new Dataloader<string, Location>(async keys => {
-    const rows = await sql<WithKey<Location>[]>`
+  new Dataloader<string, Location & Geofence>(async keys => {
```

Comment:
shouldnt need `Location & Geofence` right? since `type Location` below has a `geofence` property. or maybe im misreading that bit

Reply:
It was needed for the `LocationResolvers` to pick up the geofence types
![Screenshot 2024-11-12 at 10 56 40 AM](https://github.com/user-attachments/assets/2e76e2ae-37ca-4920-b706-e9d136a7c25b)